### PR TITLE
Benine thumbnails

### PR DESCRIPTION
**Pull request by bart-de-jonge**
Added thumbnails of presets in timelineblocks, timeline toast and started work in timeline dialog

### CODE REVIEW

**Comment by Markvanderuit** [comment]
Nothing to comment, looks good to me. I'll have a look at the scaling issues tomorrow as well.

**Comment by Markvanderuit** [comment]
Also, very nice catch with the bugfix.

File: elements/timeline-view/timeline-block.html [hunk 1]
```diff
@@ -59,41 +112,12 @@
 
             .bottom-size {
                 background-color: transparent;
-                height: 100%;
+                /*height: 100%;*/
```
Comment (martijnjanssen) [inline elements/timeline-view/timeline-block.html]: Commented code.

**Comment by martijnjanssen** [comment]
Nice work!

**Pull request merged**